sdk/inc/azure/core/az_mqtt5_rpc_client.h

```diff
@@ -28,20 +29,20 @@
 
 // ~~~~~~~~~~~~~~~~~~~~ Codec Public API ~~~~~~~~~~~~~~~~~
 /**
- * @brief MQTT5 RPC Client options.
+ * @brief MQTT5 RPC Client Codec options.
  *
  */
 typedef struct
 {
   /**
    * @brief Timeout in seconds for subscribing (must be > 0).
    */
-  uint32_t subscribe_timeout_in_seconds;
+  int32_t subscribe_timeout_in_seconds;
 
   /**
    * @brief Timeout in seconds for publishing (must be > 0).
    */
-  uint32_t publish_timeout_in_seconds;
+  int32_t publish_timeout_in_seconds;
```

Comment:
Why to signed?

Reply:
Just as a safety measure in case they overflow it's easier to detect